java/src/lib.rs

```diff
@@ -89,10 +79,59 @@ fn redis_value_to_java<'local>(
             data: _,
             attributes: _,
         } => todo!(),
-        Value::Push { kind: _, data: _ } => todo!(),
+        // Create a java `Map<String, Object>` with two keys:
+        //   - "kind" which corresponds to the push type, stored as a `String`
+        //   - "values" which corresponds to the array of values received, stored as `Object[]`
```

Comment:
If we only have 2 keys, why not just return an array of 2 elements? Not a big deal, but I'm curious why we went with a Map.

Reply:
Good idea, but I'm following python client.